CONTRIBUTING.md

```diff
@@ -4,55 +4,30 @@ Thank you for your interest in contributing to Ice!
 
 ## How to Contribute
 
-Follow these steps to contribute code or documentation to this project:
+To contribute code or documentation to this project, please follow these steps:
 
-1. Download and review carefully our [Contributor Agreement](https://zeroc.com/download/ZeroCContributorAgreement.pdf), and send us a completed and signed copy.
+1. Review the [Contributor Agreement](https://gist.github.com/zcabot/1a4c24dca55adaa83d78cdeabc63226b).
```

Comment:
You should call it the ZeroC Contributor Agreement like in the agreement itself.